appveyor.yml

```diff
@@ -0,0 +1,36 @@
+image: Visual Studio 2017
+
+hosts:
+  example.letsencrypt.org: 127.0.0.1
+  elpmaxe.letsencrypt.org: 127.0.0.1
+
+environment:
+  DOCKER_USER:
+    secure: E0lafFBkoStDl0EOWJ7RJg==
```

Comment:
It's better to define these values (`DOCKER_USER`, `DOCKER_PASS`) inside the Appveyor interface.

![appveyor-secure-env](https://user-images.githubusercontent.com/5674651/53303149-4f9d3300-3867-11e9-89a9-0c236825949d.png)

Reply:
You are totally right ! It makes the configuration more portable, for the same level of security. I updated the code, and the PR description.